examples/Cargo.toml

```
@@ -0,0 +1,2 @@
+[workspace]
+members = ["todo"]
```

```suggestion
members = ["*"]
```

this will automatically add all directory in in the same cwd as a member

---

actually discard this, if this is done with the current directory structure, it will have a `target` directory, which would also be included (to my knowledge) messing the compile up or cargo complaining